File: Python/Array/1833_Maximum Ice Cream Bars.py
class Solution:
    def maxIceCream(self, costs: List[int], coins: int) -> int:
        
        
        costs.sort()
         
        
        count = 0
        
        for i in costs:
            if i > coins:
                return count
            else:
                count+=1
                coins -= i
        return count

File: C++/Array/1833_Maximum Ice Cream Bars.cpp
class Solution {
public:
    int maxIceCream(vector<int>& costs, int coins) {
        sort(begin(costs), end(costs));
            
        int count = 0;
        
        for(auto it:costs){
            if(it > coins){
                return count;
            }
            else{
                coins -= it;
                    
                count++;
            }
        }
        
        return count;
    }
};